src/chapters/radius-auth.xml

```diff
@@ -0,0 +1,240 @@
+                        parameter is required and defaults to null if not specified.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-auth-protocol</property></term>
+                <listitem>
+                    <para>The authentication protocol to use when talking to the RADIUS server.
+                        This parameter is required for the extension to operate.  Supported
+                        values are: pap, chap, mschapv1, mschapv2, eap-md5, eap-tls, and eap-ttls.
+                        Support for PEAP is implemented inside the extension, but, due to a regression
+                        in the JRadius implementation, it is currently broken.  Also, if you specify
+                        eap-ttls you will also need to specify the
+                        <property>radius-eap-ttls-inner-protocol</property> parameter in order to
+                        properly configure the protocol used inside the EAP TTLS tunnel.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-key-file</property></term>
+                <listitem>
+                    <para>The combination certificate and private key pair to use for TLS-based
+                        RADIUS protocols that require a client-side certificate.  The file should be
+                        present in the GUACAMOLE_HOME directory, and should be readable by the user
+                        running Tomcat (or whatever Java servlet container you're running).  If not
+                        specified, this defaults to radius.pem.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                 <term><property>radius-key-type</property></term>
+                 <listitem>
+                     <para>The type of the key file specified by the <property>radius-key-file</property> 
+                        parameter.  If not specified, this defaults to pkcs12, the default used by 
+                        the JRadius library.</para>
+                 </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-key-password</property></term>
+                <listitem>
+                    <para>The password of the private key specified in the 
+                        <property>radius-key-file</property> parameter.  Defaults to null if not specified.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>ca-key-file</property></term>
+                <listitem>
+                    <para>The file that stores the certificate authority certificates for the
+                        connection to the RADIUS server.  Defaults to no file - if specified, the
+                        file must be present in the GUACAMOLE_HOME directory.</para>
```

Comment:
Does this mean that the entire path specified here is interpreted relative to `GUACAMOLE_HOME`? Or that this must be the name of a file which is present directly within `GUACAMOLE_HOME`?

Reply:
In fact ... it might be easier to conceptualize for users, easier to document, and cleaner to code if things are instead done with a `FileGuacamoleProperty`:

https://github.com/apache/guacamole-client/blob/798349e2310d3ba90e3a1d5fa276c2c8efc19c82/guacamole-ext/src/main/java/org/apache/guacamole/properties/FileGuacamoleProperty.java

That would tie the type of the property to `File`, rather than having to interpret the `String` manually, and would result in the path being interpreted as an absolute path. Relying on a file with a standardized filename within `GUACAMOLE_HOME` can then be left as the default.

Reply:
Okay, I will reword documentation, and then rework the extension to go this route.

Reply:
Documentation reworded, and submitted a new PR for the client to modify the extension to use `FileGuacamoleProperty`.